pkg/prompter/prompter.go

```diff
@@ -32,103 +34,40 @@ type FileReader interface {
 
 // New instantiates a new Prompter.
 func New(stdin FileReader, stdout FileWriter, stderr FileWriter) *Prompter {
+	// TODO: Enhance logic to look at configuration for prompter type.
+	prompterType := os.Getenv("GH_PROMPTER")
```

Comment:
Idea from @maaslalani : what if we forced `GH_PROMPTER` to the new glamorous prompter experience if `ACCESSIBLE=1`?

_main concern about how the user's experience with 3rd party extensions would play out when they eventually upgrade `cli/go-gh` but didn't consider the changes_ 🤔